### PR TITLE
More fixes for windows

### PR DESCRIPTION
**Pull request by GustavoLCR**
https://github.com/radare/radare2/issues/10613

### CODE REVIEW

File: new/db/formats/mangling/rust [hunk 1]
```diff
@@ -34,7 +34,7 @@ EXPECT=<<EOF
 <Test + 'static as foo::Bar<Test>>::bar
 EOF
 CMDS=<<EOF
-"!rabin2 -D rust _ZN71_\$LT\$Test\$u20\$\$u2b\$\$u20\$\$u27\$static\$u20\$as\$u20\$foo..Bar\$LT\$Test\$GT\$\$GT\$3barE"
+!rabin2 -D rust _ZN71_\$LT\$Test\$u20\$\$u2b\$\$u20\$\$u27\$static\$u20\$as\$u20\$foo..Bar\$LT\$Test\$GT\$\$GT\$3barE
```
Comment (radare) [inline new/db/formats/mangling/rust]: Shouldnt this thing run with “” too?

Comment (GustavoLCR) [reply new/db/formats/mangling/rust]: It works without quotes, even on linux.

**Comment by radare** [comment]
Whats the problem with the quotes on windows?

**Comment by GustavoLCR** [comment]
> Whats the problem with the quotes on windows?

Its not the quotes, its that the $ doesn't need to be escaped, and r2r only removes the \ from the command if it starts with a !

**Pull request merged**